Sources/GRPCHTTP2Core/Server/GRPCServerStreamHandler.swift

```diff
@@ -37,7 +37,7 @@ final class GRPCServerStreamHandler: ChannelDuplexHandler {
   private var pendingTrailers:
     (trailers: HTTP2Frame.FramePayload, promise: EventLoopPromise<Void>?)?
 
-  private let methodDescriptorPromise: EventLoopPromise<MethodDescriptor>
+  internal let methodDescriptorPromise: EventLoopPromise<MethodDescriptor>
```

Comment:
Can we keep this private? We pass it in to the handler so we already have a handle on it.